tensorboard/pip_package/build_pip_package.sh

```diff
@@ -43,6 +52,17 @@ function main() {
 
   pushd ${TMPDIR} >/dev/null
   rm -f MANIFEST
+
+  echo $(date) : "=== Activating virtualev ==="
+  virtualenv "${TMPVENVDIR}"
```

Comment:
Just for my understanding, why'd we activate a virtual env for installation?

Reply:
We activated a virtualenv so that we could install setuptools 36.2.0+ as described below (and oops, there's a typo in the comment, I will fix that).  I wouldn't want the pip package build script to actually update the system setuptools, and it seems nicer to just use a virtualenv rather than erroring out and requiring the user to either use an outer virtualenv or update the system setuptools version.